### PR TITLE
Sunhwa's basic python excercises

### PR DESCRIPTION
**Pull request by sunhwap**
I posted my python basic exercises.

http://sunhwap.github.io/spring2014/2014/01/24/sunhwapythonexercises.html

### CODE REVIEW

**Comment by gerbal** [comment]
put `http://` in front of the link to your site and it'll turn into a proper hyperlink.

**Comment by sunhwap** [comment]
Whoops!  Thanks a lot.
Sunhwa

**Comment by eah13** [comment]
@sunhwap that site doesn't work for me.  Does it for you?

See my comments in the code.

File: _posts/2014-01-23/2014-01-24-sunhwapythonexercises.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+author: Sunhwa
+title: program excercise
+date: 01-22-14
```
Comment (eah13) [inline _posts/2014-01-23/2014-01-24-sunhwapythonexercises.md]: This date is not in a format that Jekyll understands.

File: _posts/2014-01-23/2014-01-24-sunhwapythonexercises.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+author: Sunhwa
```
Comment (eah13) [inline _posts/2014-01-23/2014-01-24-sunhwapythonexercises.md]: This author is not something that Jekyll understands.  Go look at the code from your first posts if you forgot what you called yourself in the `_config` file.  Or go look in the `_config` file.

**Comment by sunhwap** [comment]
Hi Elliot,
It doesn't.
Thanks.

File: _posts/2014-01-23/2014-01-24-sunhwapythonexercises.md (hunk 2 above)
Comment (sunhwap) [inline _posts/2014-01-23/2014-01-24-sunhwapythonexercises.md]: Thank you so much and am sorry that I made such mistakes.
Sunhwa

**Comment by eah13** [comment]
Hey alright!  This is showing up.  Way to keep at it.

**Comment by sunhwap** [comment]
Hi Elliot,Thanks you so much for your help!!!sunhwa

Date: Mon, 27 Jan 2014 15:01:06 -0800

**Pull request merged**